src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$primary-color: #007bff;
+$hover-color: #0056b3;
+$border-color: #ccc;
+$text-color: #333;
+$background-color: lightblue;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: $background-color;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  min-height: 100vh;
 }
+
+h1 {
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+.product-card {
+  width: 200px; 
+  background-color: #fff;
+  border: 1px solid $border-color;
+  border-radius: 8px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  transition: transform 0.3s ease, box-shadow 0.3s ease;
+
+  &:hover {
+    transform: translateY(-5px);
+    box-shadow: 0 5px 15px rgba(0, 0, 0, 0.1);
+  }
+
+  &__image {
+    max-width: 100%;
+    border-radius: 4px;
+    margin-bottom: 12px;
+  }
+
+  &__title {
+    font-size: 18px;
+    color: $text-color;
+    margin: 8px 0;
+  }
+
+    &__description {
+      font-size: 14px;
+      color: adjust-color($text-color, $lightness: -20%);
+      margin-bottom: 12px;
+    }
+
+  &__price {
+    font-size: 16px;
+    font-weight: bold;
+    color: $text-color;
+    margin-bottom: 16px;
+  }
+
+  &__button {
+    padding: 8px 16px;
+    background-color: $primary-color;
+    color: #fff;
+    text-decoration: none;
+    border-radius: 4px;
+    border: none;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+
+    &:hover {
+      background-color: $hover-color;
+    }
+  }
+}
```

Comment:
The `stars` BEM block is missing. According to the task requirements, this block should be included and styled in the SCSS file.